Java/Java Anagrams/Solution1.java

```diff
@@ -0,0 +1,88 @@
+
+
+
+//Two strings  and  are called anagrams if they consist same characters, but may be in different orders. So the list of anagrams of  is .
+//
+//Given two strings, print Anagrams if they are anagrams, print Not Anagrams if they are not. The strings may consist at most  English characters; the comparison should NOT be case sensitive.
+//
+//This exercise will verify that you can sort the characters of a string, or compare frequencies of characters.
+//
+//
+//
+//Sample Input 0
+//
+//anagram
+//margana
+
+//Sample Output 0
+//
+//Anagrams
+
+
+
+//Sample Input 1
+//
+//anagramm
+//marganaa
+
+//Sample Output 1:
+//
+//Not Anagrams
+
+
+
+
+
+import java.util.*;
+
+public class HelloWorld{
+
+     public static void main(String []args){
+        
+        // here i am taking two string inputs for comparison
+        String str1 = "hello";
+        String str2 ="olleh";
+        
+    
+        boolean Anagram = false;   // Firstly our anagram will be initialized to false
+        
+        boolean[] checked = new boolean[str1.length()];  // A check point is necessary so that a single letter or character cannot be comparisoned two or multiple times
```

Comment:
A check point is necessary so that a single letter or character cannot be comparisoned two or multiple times